src/js/Content/Features/Store/PointsShop/FBackgroundPreviewLink.js

```diff
@@ -0,0 +1,40 @@
+import {HTML, Localization} from "../../../../modulesCore";
+import {Feature, User} from "../../../modulesContent";
+
+export default class FBackgroundPreviewLink extends Feature {
+
+    checkPrerequisites() {
+        return User.isSignedIn;
+    }
+
+    apply() {
+
+        new MutationObserver(async mutations => {
+            for (const {addedNodes} of mutations) {
+                if (addedNodes.length !== 1 || !addedNodes[0].classList.contains("FullModalOverlay")) { continue; }
+
+                let videoEl;
+
+                await new Promise(resolve => {
+                    new MutationObserver((mutations, observer) => {
+                        videoEl = mutations[0].addedNodes[0].querySelector("[class^='redeempointsmodal_BackgroundPreviewContainer'] video");
+                        observer.disconnect();
+                        resolve();
+                    }).observe(addedNodes[0], {"childList": true, "subtree": true});
+                });
```

Comment:
Is it possible the modal content is loaded before the observer is attached? This worked while testing so I assumed no...